cpp/test/label/label.cu

```diff
@@ -59,8 +59,8 @@ TEST_F(MakeMonotonicTest, Result)
 
   ASSERT_TRUE(devArrMatch(actual.data(), expected.data(), m, raft::Compare<bool>(), stream));
 
-  delete data_h;
-  delete expected_h;
+  delete [] data_h;
+  delete [] expected_h;
```

Comment:
```suggestion
  delete[] data_h;
  delete[] expected_h;
```

looks like style checks don't want a space there: https://github.com/rapidsai/raft/actions/runs/12755374162/job/35551255311?pr=2539#step:7:1573

Applying that change to restart CI for you here.